--- xfce-utils/scripts/xinitrc~	2004-06-16 17:30:02.000000000 +0200
+++ xfce-utils/scripts/xinitrc	2004-06-17 10:48:26.862255136 +0200
@@ -92,7 +92,7 @@
 		$panel
 		ret=$?
 		while test $ret -ne 0; do
-			xmessage -center -file - -timeout 20 -title Error <<EOF
+			gxmessage -center -file - -timeout 20 -title Error <<EOF
 A crash occured in the panel
 Please report this to the xfce4-dev@moongroup.com list
 Meanwhile the panel will be restarted
